app/services/deploy_service.rb

```diff
@@ -28,6 +28,8 @@ def self.create_github_pull_request(deploy_strategy)
         group_by(&:login).map{|k,v| [v.size, k] }.sort.reverse.map(&:last)
       assignee = sorted_logins.detect { |l| client.check_assignee(repo, l) }
       client.add_assignees(repo, pr.number, assignee) if assignee
+      slack = Slack::Web::Client.new
+      slack.chat_postMessage(channel: '#dev', message: "<#{pr.html_url}|Deploy PR> ready for #{repo}", as_user: true)
```

Comment:
How should this behave if the Slack call fails? Right now I think it will cause the whole cron to exit. This project doesn't have much error-reporting in place currently, so might be better to just log a warning for now.